.github/workflows/acceptance-tests-runner.yml

```diff
@@ -419,6 +419,8 @@ jobs:
           terraform_wrapper: false    
       - name: Acceptance Tests
         env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.aws_access_key_id }}
```

Comment:
can we also add this to mig test config so they're synchronized even if no mig tests need it now?

Reply:
good catch, will add

Reply:
added